Index: src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>server.port= 8090\r\n\r\nspring.datasource.url=jdbc:mysql://localhost:3306/finalproject\r\nspring.datasource.username=root\r\nspring.datasource.password=1234\r\nspring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver\r\nspring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect\r\n\r\n#? ???\r\nspring.devtools.livereload.enabled=true\r\nspring.devtools.restart.enabled=true\r\n\r\nspring.thymeleaf.cache=false\r\n\r\n# true ??? JPA ??? ?? ??\r\nspring.jpa.show-sql=true\r\n\r\n# DDL(create, alter, drop) ??? DB? ?? ??? ??? ? ??.\r\nspring.jpa.hibernate.ddl-auto=update\r\n\r\n# JPA? ???? Hibernate? ????? ??? SQL? ???? ????.\r\nspring.jpa.properties.hibernate.format_sql=true\r\n\r\n# Thymeleaf ??? ?? ?? ?? (???: true)\r\n\r\n# Thymeleaf ???? ??? ? ??? ??\r\nspring.thymeleaf.prefix=classpath:/templates/\r\nspring.thymeleaf.suffix=.html\r\n\r\n\r\n\r\n\r\n\r\n\r\n# Thymeleaf ??? ??? ?? ?? (???: HTML)\r\nspring.thymeleaf.mode=HTML\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/src/main/resources/application.properties b/src/main/resources/application.properties
--- a/src/main/resources/application.properties	(revision 3a00665e808ba7a1d8253f9f0a948dd1865418b6)
+++ b/src/main/resources/application.properties	(date 1693136202000)
@@ -1,4 +1,4 @@
-server.port= 8090
+server.port= 8080
 
 spring.datasource.url=jdbc:mysql://localhost:3306/finalproject
 spring.datasource.username=root
Index: src/main/resources/templates/tourism/tourSearch.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/templates/tourism/tourSearch.html b/src/main/resources/templates/store/tourism/tourSearch.html
rename from src/main/resources/templates/tourism/tourSearch.html
rename to src/main/resources/templates/store/tourism/tourSearch.html
--- a/src/main/resources/templates/tourism/tourSearch.html	(revision 3a00665e808ba7a1d8253f9f0a948dd1865418b6)
+++ b/src/main/resources/templates/store/tourism/tourSearch.html	(date 1693136478000)
@@ -39,15 +39,7 @@
                             <button class="modal_btn sel" onclick="insertAreaCode(this)" value="">전체</button>
                         </div>
                         <div class="btn_list sigungu_list">
-                            <!--<button class="modal_btn sigungu_btn">강남구</button>
-                            <button class="modal_btn sigungu_btn">강북구</button>
-                            <button class="modal_btn sigungu_btn">강서구</button>
-                            <button class="modal_btn sigungu_btn">강동구</button>
-                            <button class="modal_btn sigungu_btn">강동구</button>
-                            <button class="modal_btn sigungu_btn">강동구</button>
-                            <button class="modal_btn sigungu_btn">강동구</button>
-                            <button class="modal_btn sigungu_btn">강동구</button>
-                            <button class="modal_btn sigungu_btn">강동구</button>-->
+
                         </div>
                     </li>
                 </ul>
diff --git a/src/main/resources/templates/tourism/tourDetail.html b/src/main/resources/templates/store/tourism/tourDetail.html
rename from src/main/resources/templates/tourism/tourDetail.html
rename to src/main/resources/templates/store/tourism/tourDetail.html
diff --git a/src/main/resources/templates/flight/flights.html b/src/main/resources/templates/store/flight/flights.html
rename from src/main/resources/templates/flight/flights.html
rename to src/main/resources/templates/store/flight/flights.html
diff --git a/src/main/resources/templates/tourism/cities.html b/src/main/resources/templates/store/tourism/cities.html
rename from src/main/resources/templates/tourism/cities.html
rename to src/main/resources/templates/store/tourism/cities.html
diff --git a/src/main/resources/templates/tourism/tourmain.html b/src/main/resources/templates/store/tourism/tourmain.html
rename from src/main/resources/templates/tourism/tourmain.html
rename to src/main/resources/templates/store/tourism/tourmain.html
